test/topology/conftest.py

```diff
@@ -48,6 +52,44 @@ def run_async(self, *args, **kwargs) -> asyncio.Future:
 Session.run_async = run_async
 
 
+# "cql" fixture: set up client object for communicating with the CQL API.
+# The host/port combination of the server are determined by the --host and
+# --port options, and defaults to localhost and 9042, respectively.
+# We use scope="session" so that all tests will reuse the same client object.
+@pytest.fixture(scope="session")
+def cql(request):
```

Comment:
Using the existing `cql` fixture from test/conftest.py doesn't work? Wouldn't the server just ignore the auth data with AllowAll?

Reply:
Like Cato the Elder, I want to state again my belief that the top-level test/conftest.py should not exist. There can be common libraries that many test suites reuse, but there will always be differences. I think this is becoming more obvious now that you have a "cql" fixture in both test/conftest.py and test/topology/conftest.py and nominally both apply and you need to start guessing which one is relevant.

Reply:
OK, if the end-game is that we remove the top-level one, then it makes sense.